soh/soh/z_message_OTR.cpp

```diff
@@ -187,4 +187,142 @@ extern "C" void OTRMessage_Init()
         CustomMessage("Hey! Hey!&You can't take the rod out of here!&I'm serious!^Do you want to quit?&\x1B&%gYes&No%w",
                       "Hey! Hey!&Du kannst die Angel doch nicht&einfach mitnehmen!&Ganz im Ernst!^Möchtest du aufhören?&\x1B&%gJa&Nein%w", //TODO Used AI translation as placeholder
                       "Holà! Holà!&Les cannes ne sortent pas d'ici!&Je suis sérieux!^Voulez-vous arrêter?&\x1B&%gOui&Non%w")); //TODO Used AI translation as placeholder
-}
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_GREETING,
+        CustomMessage("Hey there, @! You&making a deposit? Or are you&here to make a withdrawal?",
+                    "Hallo, @! Machen Sie&eine Einzahlung? Oder sind Sie&hier, um eine Auszahlung&vorzunehmen?",
+                    "Salut, @! Vous&effectuez un dépôt ? Ou êtes-vous&ici pour effectuer un retrait?"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_OPTIONS,
+        CustomMessage("What would you like to do?" "\x1C" "&%gDeposit Rupees&%gWithdrawal Rupees&%gNothing%w",
+                    "Was würdest du gern tun?" "\x1C" "&%gRubine einzahlen&%gRubine abheben&%gNichts%w",
+                    "Qu'est-ce que tu aimerais faire?" "\x1C" "&%gDéposer des rubis&%gRetirer des rubis&%gRien%w"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_BALANCE,
+        CustomMessage("If I remember, as of your last &warp transfer your balance is&%y{{playerBalance}}%w rupees.",
+                      "Wenn ich mich erinnere, betrug Ihr&Guthaben bei Ihrem letzten&Warp-Transfer %y{{playerBalance}}%w Rupien.",
+                      "Si je me souviens bien, lors de&votre dernier transfert Warp,&votre solde était de %y{{playerBalance}}%w roupies."));
+    CustomMessageManager::Instance->CreateMessage(
+            customMessageTableID, TEXT_BANKER_WITHDRAWAL_AMOUNT,
+            CustomMessage("\x08Select Withdrawal Amount:&     rupees.&Set the amount with \xAA and&press \x9F to decide.\x0A",
+                        "\x08" "Auszahlungsbetrag:&     Rubine.&Wählen Sie den Betrag mit \xAA.&Drücken Sie \x9F, um zu entscheiden.\x0A",
+                        "\x08Sélectionnez le montant du retrait:&     Roupies.&Réglez le montant avec \xAA et&appuyez sur \x9F pour décider.\x0A"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_WITHDRAWAL_CONFIRM,
+        CustomMessage("Here are your rupees. Your new&balance is %y{{playerBalance}}%w rupees. See you&later!",
+                      "Hier sind deine Rupien. Ihr neues&Guthaben beträgt %y{{playerBalance}}%w Rupien.&Bis später!",
```

Comment:
Considering the majoritarian usage of the polite form in the added dialogue lines, I would suggest to standardize the German dialogue in a consistent way.

`Hier sind deine Rupien.` &#8594; `Hier sind Ihre Rupien.`